### PR TITLE
QOL improvements

### PR DESCRIPTION
**Pull request by MohanVashist1**
Quality of life improvements.
+ updated Readme
+ Used pathlib instead of os for common operations
+ added ability to save files
+ comment updating

### CODE REVIEW

File: output/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```
Comment (SpencerMcMurray) [inline output/.gitignore:1]: just use the */ format in the main gitignore

File: helpers.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
-from scipy.signal.windows import gaussian
-from scipy.ndimage import binary_dilation
 from random import randint
+
 import numpy as np
+from scipy.ndimage import binary_dilation
+from scipy.signal.windows import gaussian
+
 import data
```
Comment (SpencerMcMurray) [inline helpers.py:9]: deadass?

**Review by SpencerMcMurray — CHANGES_REQUESTED** [review]

File: helpers.py (hunk 2 above)
Comment (MohanVashist1) [reply helpers.py:9]: they were already used, just organized the imports

File: output/.gitignore (hunk 1 above)
Comment (MohanVashist1) [reply output/.gitignore:1]: cant, otherwise the directory would not be committed because there would be no files in the directory

**Pull request merged**